crates/sui/Cargo.toml

```diff
@@ -90,3 +92,15 @@ normal = ["jemalloc-ctl"]
 name = "generate-genesis-checkpoint"
 path = "src/generate_genesis_checkpoint.rs"
 test = false
+
+[[bin]]
+name = "sui"
+path = "src/main.rs"
```

Comment:
Are the `bin` declarations still needed? This one looks like it exactly matches what the default would be.